agent/handlers/task_server_setup_sudo_integ_test.go

```diff
@@ -0,0 +1,427 @@
+	family                   = "sleep"
+	version                  = "1"
+	eniIPv4Address           = "10.0.0.2"
+	macAddress               = "06:96:9a:ce:a6:ce"
+	privateDNSName           = "ip-172-31-47-69.us-west-2.compute.internal"
+	subnetGatewayIpv4Address = "172.31.32.1/20"
+	taskCredentialsID        = "taskCredentialsId"
+	defaultIfname            = "eth0"
+	clusterName              = "default"
+	availabilityzone         = "us-west-2b"
+	vpcID                    = "test-vpc-id"
+	containerInstanceArn     = "containerInstanceArn-test"
+	startEndpoint            = "/api/%s/fault/v1/%s/start"
+	stopEndpoint             = "/api/%s/fault/v1/%s/stop"
+	statusEndpoint           = "/api/%s/fault/v1/%s/status"
+	defaultNamespace         = "host"
+	requestMaxRetries        = 3
+	requestTimeoutInSeconds  = 5 * time.Second
+
+	port               = 4321
+	protocol           = "tcp"
+	trafficType        = "ingress"
+	delayMilliseconds  = 100
+	jitterMilliseconds = 200
+	lossPercent        = 6
+
+	serverPort = 3333
+)
+
+var (
+	now = time.Now()
+
+	ipSources         = []string{"1.1.1.1", "2.2.2.2"}
+	ipSourcesToFilter = []string{"8.8.8.8"}
+
+	happyBlackHolePortReqBody = map[string]interface{}{
+		"Port":        port,
+		"Protocol":    protocol,
+		"TrafficType": trafficType,
+	}
+
+	happyNetworkLatencyReqBody = map[string]interface{}{
+		"DelayMilliseconds":  delayMilliseconds,
+		"JitterMilliseconds": jitterMilliseconds,
+		"Sources":            ipSources,
+		"SourcesToFilter":    ipSourcesToFilter,
+	}
+
+	happyNetworkPacketLossReqBody = map[string]interface{}{
+		"LossPercent":     lossPercent,
+		"Sources":         ipSources,
+		"SourcesToFilter": ipSourcesToFilter,
+	}
+
+	container = &apicontainer.Container{
+		Name:                containerName,
+		Image:               imageName,
+		ImageID:             imageID,
+		DesiredStatusUnsafe: apicontainerstatus.ContainerRunning,
+		KnownStatusUnsafe:   apicontainerstatus.ContainerRunning,
+		CPU:                 cpu,
+		Memory:              memory,
+		Type:                apicontainer.ContainerNormal,
+		ContainerArn:        "arn:aws:ecs:ap-northnorth-1:NNN:container/NNNNNNNN-aaaa-4444-bbbb-00000000000",
+		KnownPortBindingsUnsafe: []apicontainer.PortBinding{
+			{
+				ContainerPort: containerPort,
+				Protocol:      apicontainer.TransportProtocolTCP,
+			},
+		},
+	}
+
+	dockerContainer = &apicontainer.DockerContainer{
+		DockerID:   containerID,
+		DockerName: containerName,
+		Container:  container,
+	}
+
+	containerNameToDockerContainer = map[string]*apicontainer.DockerContainer{
+		taskARN: dockerContainer,
+	}
+
+	association = apitask.Association{
+		Containers: []string{containerName},
+		Content: apitask.EncodedString{
+			Encoding: associationEncoding,
+			Value:    associationValue,
+		},
+		Name: associationName,
+		Type: associationType,
+	}
+
+	agentStateExpectations = func(state *mock_dockerstate.MockTaskEngineState) {
+		task := standardFaultInjectionTask()
```

Comment:
nit: Probably better to rename this file to something fault-injection specific since we are defining global variables that only make sense for fault injection tests.

Reply:
sure I will rename this file

Reply:
moved this file into ecs-agent directory and renamed it to be `handlers_sudo_integ_test.go`